Fix "OE fails to build because of CMake dependency on libnl-tiny"

Upstream has a stripped down version of libnl-3 called libnl-tiny
that OpenEmbedded doesn't have, therefore we need to modify the
CMakeLists.txt to properly handle the libnl-3 library instead
f libnl-tiny.

Signed-of-by: "Daniel F. Dickinson <cshored@thecshore.com>"
Index: git/CMakeLists.txt
===================================================================
--- git.orig/CMakeLists.txt
+++ git/CMakeLists.txt
@@ -8,10 +8,10 @@ set(CMAKE_SHARED_LIBRARY_LINK_C_FLAGS ""
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -std=c99")
 
 FIND_PATH(ubox_include_dir libubox/uloop.h)
-FIND_PATH(libnl-tiny_include_dir netlink-generic.h PATH_SUFFIXES libnl-tiny)
-INCLUDE_DIRECTORIES(${ubox_include_dir} ${libnl-tiny_include_dir})
+FIND_PATH(libnl3_include_dir netlink/msg.h PATH_SUFFIXES libnl3)
+INCLUDE_DIRECTORIES(${ubox_include_dir} ${libnl3_include_dir})
 
-FIND_LIBRARY(libnl NAMES nl-tiny)
+FIND_LIBRARY(libnl NAMES nl-3)
 
 add_definitions(-D_GNU_SOURCE -Wall -Werror -Wextra)
 
